contrib/babelfishpg_tsql/sql/linked_servers_tsql.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE sys.babelfish_server_options (
```

Comment:
as we are adding a new catalog table, we should also add new rules to metadata consistency checker

Reply:
Added it